confdgnmi/src/confd_gnmi_client.py

```diff
@@ -131,15 +137,22 @@ def read_subscribe_responses(responses, read_count=-1):
         log.info("==> read_count=%s", read_count)
         # example to cancel
         # responses.cancel()
-        for response in responses:
-            log.info("******* Subscription received response=%s read_count=%i",
-                     response, read_count)
-            print("subscribe - response read_count={}".format(read_count))
-            ConfDgNMIClient.print_notification(response.update)
-            if read_count > 0:
-                read_count -= 1
-                if read_count == 0:
-                    break
+        try:
+            for response in responses:
+                log.info("******* Subscription received response=%s read_count=%i",
+                         response, read_count)
+                print("subscribe - response read_count={}".format(read_count))
+                ConfDgNMIClient.print_notification(response.update)
+                if read_count > 0:
+                    read_count -= 1
+                    if read_count == 0:
+                        break
+        except _MultiThreadedRendezvous as e:
+            if "EOF" in e.details():
+                log.warning("e.details=%s", e.details())
+            else:
+                raise e
+        log.info("Stopped reading SubscribeResponse(s)")
         log.info("Canceling read")
```

Comment:
"canceling" sounds like non-positive result, maybe change log message?

Reply:
changed to log.debug("Notifying server (with cancel() ) responses read.")